.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+<!--  
+Provide a high-level description of this change.   
+-->
```

Comment:
Does it make sense to ask for an Overview and a Changelog separately, given that most likely there will be redundancy between the two?

Reply:
Changelogs are mostly 1 liners, while overviews can easily be extended to a paragraph. Let's leave it as is, for now.